pages/admin/PersonManagementView.page.ts

```diff
@@ -13,6 +13,10 @@ export class PersonManagementViewPage{
     readonly table_header_Rolle: Locator;
     readonly table_header_Zuordnungen: Locator;
     readonly table_header_Klasse: Locator;
+    readonly comboboxDown_Schule: Locator;
```

Comment:
```suggestion
    readonly comboboxMenuIcon_Schule: Locator;
```
Oder ähnlich. Bitte überall anpassen